### PR TITLE
renamed html file to index.html.

### PR DESCRIPTION
**Pull request by PaulHowley**
Couldn't do a pull request with up to date changes so created new branch and renamed index file anyway. Screenshots included below:
screen shot 2019-01-23 at 14 33 14
screen shot 2019-01-23 at 14 33 40
screen shot 2019-01-23 at 14 33 59
screen shot 2019-01-23 at 14 32 39

### CODE REVIEW

**Comment by krisquigley** [comment]
I would look at trying to get the 3 elements evenly distributed amongst the page.  This may require editing the images so that they are the same size (some might need adding white space around them)

**Review by krisquigley — COMMENTED** [review]
Only the file has been renamed, so I'm unable to comment on your code

**Comment by PaulHowley** [comment]
Kris, see latest update. I've tidied up the code so you should be able to see most of it and I've re-sized the images to be the same size. Screenshots below:
image


image

image

image

File: index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    <meta name='viewport' content='width=device-width, initial-scale=1.0'>
+    <link rel="stylesheet" href="Styles/main.css">
+  </head>
+  <body>
+    <div class='c-container'>
+      <div class='c-container__banner'>
+          <h1>Play the Rock Paper Scissors Game!</h1>
+      </div>
+      <div class='c-container__main'>
+        <div class="c-main__box-1">
```
Comment (krisquigley) [inline index.html]: You could wrap these 3 inside a div and just hide that one div, rather than hiding each image individually

File: index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Mobile responsive design</title>
+    <meta charset='utf-8'>
+    <meta name='viewport' content='width=device-width, initial-scale=1.0'>
+    <link rel="stylesheet" href="Styles/main.css">
```
Comment (krisquigley) [inline index.html]: Always use lowercase folders and filenames

**Review by krisquigley — COMMENTED** [review]
Looking good.  Now let's refactor the querySelectors to not use elements or classes, as these can change.

Instead we'll use data-attributes. So we can add `<button data-behavior="playerSelection">Text</button>`

Then for the query selector: 
```
document.querySelectorAll("button[data-behavior='playerSelection']")
```

Remember to always use american spelling in web dev

File: Scripts/script.js [hunk 3]
```diff
@@ -1,85 +1,78 @@
...
           result = `Player Wins! ${playerSelection} beats ${computerSelection}.`
           playerScore++
-        } else {
+  } else {
           result = 'You must choose rock, paper or scissors'
         }
 
-        const resultsText = document.querySelector('.c-div__results--text1')
-    
+  const resultsText = document.querySelector('.c-div__results--text1')
```
Comment (krisquigley) [inline Scripts/script.js]: These will need updating as well as all querySelectors

File: Scripts/script.js [hunk 4]
```diff
@@ -1,85 +1,78 @@
...
+        currentScores.textContent = `Final Scores: Player = ${playerScore}, Computer = ${computerScore}. Game over the Computer is the first to 5. Commiserations!`
+      }
+      const imagesHidden = document.querySelector('.c-main__boxHolder')
+            imagesHidden.style.display = 'none'
 
-          /* const main = document.querySelector('.main')
-          const resetButton = document.createElement('button')
-          resetButton.type = 'button'
-          resetButton.textContent = 'Click here to reset' */
+   // const rockHidden = document.querySelector('.c-box-1__rock-img')
```
Comment (krisquigley) [inline Scripts/script.js]: If these are no longer needed, then they can be removed

File: Scripts/script.js [hunk 5]
```diff
@@ -1,85 +1,78 @@
...
-      }
-        const gameReset = document.querySelector('.reset_button')
-        gameReset.addEventListener('click', () => {
+    playerScore = 0
+    computerScore = 0
+  }
+}
+        
+const gameReset = document.querySelector("button[data-behavior='resetGame']")
+      gameReset.addEventListener('click', () => {
```
Comment (krisquigley) [inline Scripts/script.js:67]: padding

File: Styles/main.css [hunk 6]
```diff
@@ -57,44 +89,32 @@ p {
...
-  .main {
+  .c-container__main {
     flex-direction: column;
   }
+
+  .c-main__boxHolder {
+    flex-direction: column;
+  }
+
   input {
```
Comment (krisquigley) [inline Styles/main.css:103]: padding

**Comment by krisquigley** [comment]
Spacing between the weapons looks spot on now!

File: index.html [hunk 7]
```diff
@@ -26,8 +26,8 @@ <h1>Play the Rock Paper Scissors Game!</h1>
         <button data-behavior='resetGame' class="c-main__reset-button" value="reset">Click here to reset the game</button>
       </div>
       <footer>
-        <p class='c-div__results--text1'>Click your selection to start.</p>
-        <p class='c-div__results--text2'>First to 5 wins. Good Luck!</p>
+        <p data-behaviour='resultsText1' class='c-div__results--text1'>Click your selection to start.</p>
+        <p data-behaviour='resultsText2'class='c-div__results--text2'>First to 5 wins. Good Luck!</p>
```
Comment (krisquigley) [inline index.html]: Ensure we use american spellings, these should all be `data-behavior`.  Once these are updated go ahead and merge

**Review by krisquigley — APPROVED** [review]

**Pull request merged**